packages/dx-grid-core/package.json

```diff
@@ -35,7 +36,7 @@
     "test:coverage": "jest --coverage",
     "build": "rollup -c rollup.config.js",
     "build:watch": "rollup -c rollup.config.js -w",
-    "lint": "eslint \"src/**\"",
+    "lint": "tslint -p tsconfig.lint.json",
     "lint:fix": "yarn run lint -- --fix"
   },
   "devDependencies": {
```

Comment:
Should we remove from the `devDependencies` `babel` and `eslint` related plugins and add `ts` related plugins such as `ts-jest`, `tslint`, `tslint-config-airbnb`, `typescript`?